problem_builder/mentoring.py

```diff
@@ -131,6 +131,12 @@ class MentoringBlock(XBlock, StepParentMixin, StudioEditableXBlockMixin, StudioC
         default=_("Mentoring Questions"),
         scope=Scope.settings
     )
+    feedback_label = String(
+        display_name=_("Feedback Messages Title"),
```

Comment:
This is super awkward. I'm not sure how it's used yet until you have a sandbox but I'm going to suggest something different.